docs/open-api/openapi.yaml

```diff
@@ -477,7 +477,21 @@ components:
           - "TOPIC"
           - "ROLE"
           - "METALAKE"
-
+    metadataObjectTypeWithoutColumn:
```

Comment:
The parameter name is a little bit weird, but acceptable.

Reply:
Rethinking this change ... maybe `metadataObjectTypeWithOwner` is better?

Reply:
Ok